frontend/public/models/yaml-templates.ts

```diff
@@ -797,4 +797,17 @@ spec:
   machineSelector:
     matchLabels:
       node-role.kubernetes.io/master: ""
+`).setIn([referenceForModel(k8sModels.ConsoleCLIToolModel), 'default'], `
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLITool
+metadata:
+  name: example
+spec:
+  name: examplecli
+  description: |
+    This is an examplecli tool decription.
```

Comment:
I'd mention here that it can have markdown content (maybe use some simple markdown as an example?)